bokehjs/src/lib/core/util/math.ts

```diff
@@ -12,14 +12,15 @@ export function angle_dist(lhs: number, rhs: number): number {
   return Math.abs(angle_norm(lhs-rhs))
 }
 
-export function angle_between(mid: number, lhs: number, rhs: number, direction: "anticlock" | "clock"): boolean {
+export function angle_between(mid: number, lhs: number, rhs: number, direction: 1 | 0): boolean {
+  // 1 = anticlock, 0 = clock
```

Comment:
Use an enum to make the API more meaningful (https://www.typescriptlang.org/docs/handbook/enums.html).

Reply:
So its preferable to pass in a string with one of "anticlock" or "clock"? In that case I can change how the value is being passed into "angle_between"

Reply:
@anthonydouc not strings, TypeScript supports actual enum variables, see e.g. https://github.com/bokeh/bokeh/blob/master/bokehjs/src/lib/models/tools/gestures/range_tool.ts#L11